README.md

```diff
@@ -55,6 +55,8 @@ regular `ElementCriteriaModel`s. For example, to list submissions to the
 ```
 
 ## Updates
+* 1.7.0
```

Comment:
Do we want to remove this section of the readme?
Otherwise we're needing to update both the (newly created) changelog, plus here.

Reply:
Removed them!